README.md

```diff
@@ -4,6 +4,20 @@
 
 A collection of React components
 
+# Installation
+
+```jsx
+npm install spartak-ui
+```
+
+or
+
+```jsx
+yarn add spartak-ui
+```
```

Comment:
For the sake of consistency, it would be great if we could include subheadings using ### before each command, indicating the name of the respective package manager. Additionally, to ensure correct syntax highlighting, it would be appreciated if you could replace jsx with bash for the shell commands.

We can update the section as follows:

## Installation

### npm

```bash
npm install spartak-ui
```

### yarn

```bash
yarn add spartak-ui
```

Thank you for your attention to these details.

Reply:
@shdq : Implemented this change. Please review